package.json

```diff
@@ -49,6 +49,7 @@
     "ember-cli-inject-live-reload": "^1.3.0",
     "ember-cli-moment-shim": "0.7.1",
     "ember-cli-qunit": "^1.1.0",
+    "ember-cli-release": "0.2.8",
```

Comment:
The `publish: true` option was added in https://github.com/lytics/ember-cli-release/pull/39 and is only available in 1.0.0-beta.0.  You may want to update the version here to the beta or call `runCommand('npm publish')` in the `config/release.js`...